Library/Formula/libmill.rb

```diff
@@ -0,0 +1,45 @@
+class Libmill < Formula
+  desc "Go-style concurrency in C"
+  homepage "http://libmill.org/"
+  url "http://libmill.org/libmill-0.16-beta.tar.gz"
+  sha256 "c6d68d8291dadf2ee111f6ceed057f2fd267e7dc334bef8d15b6316c35b6a0ca"
+  head "https://github.com/sustrik/libmill.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "cmake" => :build
+
+  def install
+    ENV.O1  # Issue 21 at https://github.com/sustrik/libmill/issues/21
```

Comment:
Can you put this comment above and add a short description of the issue rather than just a link? Thanks!